app/src/main/java/com/eveningoutpost/dexdrip/ImportedLibraries/dexcom/Dex_Constants.java

```diff
@@ -92,8 +92,8 @@
         DOWN_45(5,"\u2198", "FortyFiveDown", -1d),
         SINGLE_DOWN(6,"\u2193", "SingleDown", -2d),
         DOUBLE_DOWN(7,"\u21CA", "DoubleDown", -3.5d),
-        NOT_COMPUTABLE(8, "", "NOT_COMPUTABLE"),
-        OUT_OF_RANGE(9, "", "OUT_OF_RANGE");
+        NOT_COMPUTABLE(8, "", "NotComputable"),
```

Comment:
Will these changes break things like `isSlopeNameInvalid()` and other string comparisons vs the friendly name?

Reply:
No, because `isSlopeNameInvalid()` [is part of](https://github.com/NightscoutFoundation/xDrip/blob/a439093a021b2183676290fbe3e726841c109372/app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java#L706) the `BgReading` class and still does not use the `TREND_ARROW_VALUES` enum.